cogs/source.py

```diff
@@ -2,17 +2,39 @@
 from discord.ext import commands
 import os
 import spacy
+from pathlib import Path
+from typing import Dict, Final, List, Any
+import yaml
 
 class SourceCog(commands.Cog):
-    """A Discord cog for replying to 'source that' message and displaying the content of the most relevant file."""
+    """
+    A Discord cog for replying to 'source that' message and displaying the content of the most relevant file.
```

Comment:
'relevant file' has no context what these files are. Also the description in general is not that good. I would be confused if I didn't know what the code did in the first place.